src/main/java/xyz/nucleoid/map_templates/MapChunk.java

```diff
@@ -66,7 +66,7 @@ public List<MapEntity> getEntities() {
     }
 
     public void serialize(NbtCompound nbt) {
```

Comment:
Suggestion: while we're making breaking changes, let's add registry context here too